frontend/simple/js/database.js

```diff
@@ -81,3 +81,16 @@ export function saveSettings (state: Object): Promise<*> {
 export function loadSettings (): Promise<Object> {
   return appSettings.getItem('testUser')
 }
+// =======================
+// Store login information
+// =======================
+const loginInfo = localforage.createInstance({
+  name: 'Group Income',
+  storeName: 'Login Information'
+})
+export function storeLogin (identity: string, hash: string): Promise<Object> {
+  return loginInfo.setItem(identity, hash)
+}
+export function retrieveLogin (identity: string): Promise<Object> {
+  return loginInfo.getItem(identity)
+}
```

Comment:
Is it possible to do this as part of the settings? The `'testUser'` above there was meant to be a reminder to use that for all user-specific settings.